.ci/lint.sh

```diff
@@ -0,0 +1,12 @@
+# binary will be $(go env GOPATH)/bin/golangci-lint
```

Comment:
Am I correct in thinking that this will try to install `golangci-lint` three times? If so, not a huge deal, but wouldn't it be better to try to run `golangci-lint`, then if that fails because it's not installed, try to install the first way and try to run, then if that doesn't work, try the second way, and so on?

Reply:
I just copied this script from avalanchego, but I'll check with Fabio.